lang/syn/src/codegen/accounts/constraints.rs

```diff
@@ -672,6 +683,63 @@ fn generate_constraint_init_group(
                 Some(fa) => check_scope.generate_check(fa),
                 None => quote! {},
             };
+            let extensions_optional_check = match extensions {
+                Some(fa) => check_scope.generate_check(fa),
```

Comment:
Nit: I think these were copied from the existing **F**reeze **A**uthority (`fa`) check, but the variable name wasn't updated for the new checks.

Same applies to other checks below.